app/src/main/assets/voronoi.py

```diff
@@ -29,11 +31,19 @@
 ]
 
 # polygonize lines and save into a file
-# each line : pair of coords of vertices for a polygon
+# each line : first two values is the input point associated to each polygon,
+# pair of coords of vertices for a polygon
```

Comment:
I need to change py script to store  the input point too (used to compute min distance)